ignite/handlers/param_scheduler.py

```diff
@@ -792,6 +792,69 @@ def simulate_values(  # type: ignore[override]
             return output
 
 
+class _CosineAnnealingWarmRestarts:
+    def __init__(self, lr_scheduler: torch.optim.lr_scheduler.CosineAnnealingWarmRestarts):
+        self._lr_scheduler = lr_scheduler
+
+    @property
+    def last_epoch(self) -> int:
+        return self._lr_scheduler.last_epoch
+
+    @property
+    def optimizer(self) -> torch.optim.Optimizer:
+        return self._lr_scheduler.optimizer
+
+    @last_epoch.setter
+    def last_epoch(self, value: int):
+        self._lr_scheduler.last_epoch = value
```

Comment:
Right now, mypy checks are failing here, see https://stackoverflow.com/questions/58121030/how-can-i-use-property-setters-and-make-mypy-happy how to make it happy.
You can test mypy locally with https://github.com/pytorch/ignite/blob/master/CONTRIBUTING.md#run-mypy-checks

Reply:
@vfdev-5 yes, thanks for helping me with this, but how to solve this?
```
ignite/handlers/param_scheduler.py:820: error: Unsupported operand types for * ("int" and "None")  [operator]
                    self._lr_scheduler.T_i = self._lr_scheduler.T_i * self._lr_scheduler.T_mult
```
I can't annotate like this: `self._lr_scheduler.T_mult: int`

Reply:
update